modules/oadp-timeouts.adoc

```diff
@@ -0,0 +1,194 @@
+ name: <backup_name>
+spec:
+ csiSnapshotTimeout: 10m
+  ...
+----
+
+[id="Velero-default-item-operation-timeout_{context}"]
+== Velereo default item operation timeout
+
+.Description
+`defaultItemOperationTimeout` defines how long to wait on asynchronous `BackupItemActions` and `RestoreItemActions` to complete before timing out. The default value is 1h.
+
+.Implementation
+.. Use this timeout only with Data Mover 1.2.x.
+.. Use this timeout to specify the amount of time a particular backup or restore should wait for the Asynchronous actions to complete. In the context of OADP features, this value is used for the Asynchronous actions involved in the Container Storage Interface (CSI) Data Mover feature.
+.. When `defaultItemOperationTimeout` is defined in the DPA using the `defaultItemOperationTimeout`, it applies to both backup and restore operations. You can use `itemOperationTimeout` to define only the backup or only the restore of those CRs, as described below in *Item operation timeout - restore*, and *Item operation timeout - backup*.
+
+.Procedure
+* Edit the values in the `spec.configuration.velero.defaultItemOperationTimeout` block of the `DataProtectionApplication` CR manifest, as in the following example:
+[source,yaml]
+----
+apiVersion: oadp.openshift.io/v1alpha1
+kind: DataProtectionApplication
+metadata:
+ name: <dpa_name>
+spec:
+  configuration:
+    velero:
+      defaultItemOperationTimeout: 1h
+   ...
+----
+
+[id="Item-operation-timeout-restore_{context}"]
+== Item operation timeout - restore
+
+.Description
+`ItemOperationTimeout` specifies the time used to wait for `RestoreItemAction` operations. The default value is 1hr.
+
+.Implementation
+.. Use this timeout only with Data Mover 1.2.x.
+.. Use this timeout for Data Mover uploads and downloads to or from the `BackupStorageLocation`. If the restore action is not completed when the timeout is reached, it will be marked as failed. If Data Mover operations are failing due to timeout issues, because of large volume sizes, then this timeout setting may need to be increased.
+
+.Procedure
+* Edit the values in the `Restore.spec.itemOperationTimeout` block of the `Restore` CR manifest, as in the following example:
+[source,yaml]
+----
+apiVersion: velero.io/v1
+kind: Restore
+metadata:
+ name: <restore_name>
+spec:
+ itemOperationTimeout: 1h
+  ...
+----
+
+[id="Item-operation-timeout-backup_{context}"]
+== Item operation timeout - backup
+
+.Description
+`ItemOperationTimeout` specifies the time used to wait for asynchronous
+`BackupItemAction` operations The default value is 1hr.
+
+.Implementation
+.. Use this timeout only with Data Mover 1.2.x.
+.. Use this timeout for Data Mover uploads and downloads to or from the `BackupStorageLocation`. If the backup action is not completed when the timeout is reached, it will be marked as failed. If Data Mover operations are failing due to timeout issues, because of large volume sizes, then this timeout setting may need to be increased.
```

Comment:
"it will be marked as failed." Passive voice. What does the marking? Also "will" is a guaranteed word, so this has to happen. Would it be best to drop "will" in numerous cases and use the present-tense of a verb? Example: X marks Y as failed (failed = a status, so 'failed').

Reply:
```suggestion
.. Use this timeout for Data Mover uploads and downloads to or from the `BackupStorageLocation`. If the backup action is not completed when the timeout is reached, it will be marked as failed. If Data Mover operations are failing due to timeout issues, then this timeout setting may need to be increased. For example, large volumes uploads or downloads might fail to complete in the designated time. 
```